docs/custom-metric-filter.md

```diff
@@ -0,0 +1,146 @@
+# ACM Custom Metric Filtering
+
+TL;DR: This guide explains how to adjust the custom metrics that Config Sync
+exports to Prometheus, Cloud Monitoring (formerly known as Stackdriver), and
+Monarch (Google's internal metrics aggregator).
+
+## Background
+
+When Config Sync is installed on a GKE cluster where a
+[default service account](http://cloud/kubernetes-engine/docs/tutorials/authenticating-to-cloud-platform)
+is available, or on a GKE cluster that has
+[Workload Identity](http://cloud/kubernetes-engine/docs/how-to/workload-identity)
+enabled and
+[IAM is correctly setup](https://cloud.google.com/anthos-config-management/docs/how-to/monitoring-config-sync#custom-monitoring),
+Config Sync will try to apply the Open Telemetry Collector config map that contains
```

Comment:
s/config map/ConfigMap/

Reply:
K8s docs style guide says to use the proper names of the resources when referring to them. Seems worth following even if it's not technically our style guide.
https://kubernetes.io/docs/contribute/style/style-guide/#api-resource-names

Reply:
Done, thanks!